### PR TITLE
build: Update OpenVINO model generation version

### PR DESCRIPTION
**Pull request by yinggeh**
#### What does the PR do?
Triton uses OpenVINO version 2024 however our models still generated with version 2023.

#### Checklist
- [x] PR title reflects the change and is of format `<commit_type>: <Title>`
- [x] Changes are described in the pull request.
- [x] Related issues are referenced.
- [x] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [x] Added [test plan](#test-plan) and verified test passes.
- [x] Verified that the PR passes existing CI.
- [x] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [ ] build

#### Test plan:
L0_openvino_models--base
L0_model_config--base

- CI Pipeline ID:
20542122

#### Related Issues: (use one of the action keywords Closes / Fixes / Resolves / Relates to)
- closes GitHub issue: #xxx

### CODE REVIEW

File: qa/common/gen_qa_models.py [hunk 1]
```diff
@@ -2486,8 +2489,7 @@ def create_fixed_models(
         import torch
         from torch import nn
     if FLAGS.openvino:
-        from openvino.inference_engine import IENetwork
-        import ngraph as ng
```
Comment (yinggeh) [inline qa/common/gen_qa_models.py:2490]: The nGraph API is deprecated and will be removed in the 2024.0 release. See https://github.com/opencv/opencv/issues/24589

**Comment by oandreeva-nv** [comment]
Is the linked pipeline up to date? The one mentioned in the PR didn't run mentioned tests

**Comment by yinggeh** [comment]
> Is the linked pipeline up to date? The one mentioned in the PR didn't run mentioned tests

Hi @oandreeva-nv. Thanks. The latest pipeline was in the slack review thread. I just updated the pipeline ID in this PR.

**Review by oandreeva-nv — APPROVED** [review]
LGTM, conditional on verifying tests
